agent/agent.go

```diff
@@ -3644,7 +3648,7 @@ func (a *Agent) loadServices(conf *config.RuntimeConfig) error {
 		}
 	}
 
-	for serviceID, _ := range persistedServiceConfigs {
+	for serviceID := range persistedServiceConfigs {
```

Comment:
`golint` told me to do this, but let me know if this should be put back. i understand wanting to hint to the reader that `_` is a possible value.